src/lib/pipeline.ml

```diff
@@ -165,13 +165,33 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
         in
         sname, Stdlib.fastq_of_input i)
 
+  (** 
+     A record to join a value and its saved version.
+
+     Previous versions of Eidisco had a [Bfx.save] construct of type
+     ['a repr -> 'a repr] which would “hide” saved values, we would
+     then put [Bfx.save ~name] calls everywhere and forget about
+     them. The new [save] from Biokepi is explicit, so to avoid too
+     much rewriting of this pipeline we pack values together.
+
+  *)
+  type 'a saved = {
+    content: 'a Bfx.repr;
```

Comment:
I've `content(s)`  used like this in other places, where a type wraps something else + some decoration, e.g.

```
utop # #show ref;;
external ref : 'a -> 'a ref = "%makemutable"                                                                                                               
type nonrec 'a ref = { mutable contents : 'a; }
```